openpype/tools/ayon_loader/ui/products_delegates.py

```diff
@@ -8,6 +9,11 @@
     VERSION_NAME_EDIT_ROLE,
     VERSION_ID_ROLE,
     PRODUCT_IN_SCENE_ROLE,
+    ACTIVE_SITE_ICON_ROLE,
+    REMOTE_SITE_ICON_ROLE,
+    REPRESENTATIONS_COUNT_ROLE,
+    SYNC_ACTIVE_SITE_AVAILABILITY,
+    SYNC_REMOTE_SITE_AVAILABILITY
```

Comment:
I don't think `SYNC_ACTIVE_SITE_AVAILABILITY` and `SYNC_REMOTE_SITE_AVAILABILITY` will be needed. You just need to hide the rows.